lib/js/fields/ArrayField.js

```diff
@@ -70,8 +70,8 @@ var ArrayField = function (_React$Component) {
      */
 
     _this.state = {
-      editing: props.formData ? props.formData.map(function () {
-        return false;
+      editing: props.formData ? props.formData.map(function (item, index) {
+        return !(0, _validation.errorSchemaIsValid)(props.errorSchema[index]);
```

Comment:
:thinking: Looks like this is from [my last PR](https://github.com/usds/us-forms-system/pull/321/files). Not sure why it's showing up as a new diff. :confused:

Reply:
Probably the PR was based off master before that PR landed?

Reply:
Oh, it's because only the /src change made it in and not the /lib one. 😢  Can you run the build and commit the FileField.js file that it updates?

Reply:
Oh nevermind, it looks like the correct FileField change is there already.